File: Trabajo1_LeyvaBanda/Tarea1.js

let nombres = ["Ana", "Luis", "Carlos", "María"];
nombres[1] = "Pedro";

nombres.forEach(n => console.log(n));

async function main() {
  const prompt = require("prompt-sync")(); 

  let tamaño = parseInt(prompt("¿Cuántos nombres quieres agregar?: "));
  let arr = [];

  for (let i = 0; i < tamaño; i++) {
    let valor = prompt(`Ingrese el nombre ${i + 1}: `);
    arr.push(valor);
  }

  console.log("Lista final:", arr);
}

main();

File: Trabajo1_LeyvaBanda/Tarea1.cs
using System;
using System.Collections.Generic;

class Program {
    static void Main() {
        // Arreglo inicial
        string[] nombres = { "Ana", "Luis", "Carlos", "María" };
        nombres[1] = "Pedro";

        foreach (string n in nombres) {
            Console.WriteLine(n);
        }

        // Lista dinámica
        Console.Write("¿Cuántos nombres quieres agregar?: ");
        int tamaño = int.Parse(Console.ReadLine());
        List<string> arr = new List<string>();

        for (int i = 0; i < tamaño; i++) {
            Console.Write($"Ingrese el nombre {i+1}: ");
            string valor = Console.ReadLine();
            arr.Add(valor);
        }

        Console.WriteLine("Lista final: " + string.Join(", ", arr));
    }
}
